.travis.yml

```diff
@@ -1,18 +1,10 @@
 language: php
 
 php:
-  - 5.4
-  - 5.5
-  - 5.6
-  - 7.0
-  - hhvm
+  - 7.1
 
 before_script:
   - composer --prefer-source install
```

Comment:
`--prefer-source` is no longer needed, as composer will simply fallback to git if communication with github's API fails